### DISCUSSION TITLE
Remove go pkg folder

### DISCUSSION THREAD

**Discussion by vianamjr** [perses/perses#1460] [Design Discussions]
In the beginning of golang we where used to use the pkg folder, but with the introduction of go modules this approach is no long necessary. There is this interesting [post](https://go.dev/doc/modules/layout) from go.dev about the project structure.

**Comment by Nexucis** [comment]
We could do that yeah. But I don't see what would be the benefit of doing that (excepting simplifying the package tree by one package). At least when you are looking to this folder/package, you should know that it can be used outside of the project.
